internal/pkg/manifest/lb_web_svc.go

```diff
@@ -157,17 +121,23 @@ func (s *LoadBalancedWebService) MarshalBinary() ([]byte, error) {
 	return content.Bytes(), nil
 }
 
-// BuildRequired returns if the service requires building from the local Dockerfile.
-func (s *LoadBalancedWebService) BuildRequired() (bool, error) {
-	return requiresBuild(s.ImageConfig.Image)
+// Port returns the exposed port in the manifest.
+// A LoadBalancedWebService always has a port exposed therefore the boolean is always true.
+func (s *LoadBalancedWebService) Port() (port uint16, ok bool) {
+	return aws.Uint16Value(s.ImageConfig.Port), true
```

Comment:
Should we have errored out somewhere if the manifest doesn't have `port` 🤔 ?

Reply:
mmm good question! I think it makes sense from the PoV of this method to assume that there is always a  `Port` for LBWS and RDWS.

I believe we always add 80 no matter what if the port is not set :)

Reply:
Hmm that is not true...If port is not set in the manifest we'll set it to `0` instead of `80`. But I believe that validation should be part of the manifest validation im going to work on.

Reply:
I asked because I remember there being a segfault when `port` wasn't set for rd web service (the user must have removed the `port` line from manifest). 

The error happened during `deploy.cloudformation.stack`'s `Template()` function. That'd mean that we didn't check for the existence of the `port` field before, and didn't set a default port either.

But yeah I agree that the validation should happens during unmarshalling/manifest validation.